CHANGELOG.md

```diff
@@ -25,6 +25,11 @@ Released: TBD
   from the `options.grammarSource` property. That property can contain arbitrary
   data,for example, path to the currently parsed file.
   [@Mingun](https://github.com/peggyjs/peggy/pull/95)
+- Parsers now can use two new functions in actions/predicates: `offset` that returns
+  just the start offset, and a `range` that returns the array `[start, end]` offsets.
+  Use them when if you don't need the whole location information, because it could be
```

Comment:
what if i need `source`?  Maybe `range()` should return the same thing as `location()` but without the line/col info, and then we don't really need `offset()`?

Reply:
Actually, I think, @felix9 wants as lightweight method as possible, so array is used, but I'm not against to change it to `{ source, start, end }` object. If even that method would too heavy, you could use `offset()`.

@felix9, do you agree with that?

Reply:
I like `{ source: "foo.txt", start: 0, end: 12}`